File: Chapter 03/Python/3.2 Coupling Exercise/Before/user_data_service.py
class UserDataService:
    def get_user_name(self, user_id):
        print("    [Service] Fetching Name...")
        return "Jane Doe"

    def get_user_email(self, user_id):
        print("    [Service] Fetching Email...")
        return "jane.doe@example.com"

    def get_user_order_ids(self, user_id):
        print("    [Service] Fetching Order IDs...")
        return ["A123", "B456"]

    def get_order_total(self, order_id):
        print(f"    [Service] Fetching Total for Order {order_id}...")
        return 99.95

File: Chapter 03/JS/3.2 Coupling Exercise/Before/userDataService.js
class UserDataService {
    getUserName(userId) {
        console.log("    [Service] Fetching Name...");
        return "Jane Doe";
    }

    getUserEmail(userId) {
        console.log("    [Service] Fetching Email...");
        return "jane.doe@example.com";
    }

    getUserOrderIds(userId) {
        console.log("    [Service] Fetching Order IDs...");
        return ["A123", "B456"];
    }

    getOrderTotal(orderId) {
        console.log(`    [Service] Fetching Total for Order ${orderId}...`);
        return 99.95;
    }
}

module.exports = UserDataService;